Assets/MixedRealityToolkit-SDK/Features/Input/Handlers/TeleportHotSpot.cs

```diff
@@ -33,8 +30,8 @@ public override void OnBeforeFocusChange(FocusEventData eventData)
 
                 if (eventData.Pointer.IsInteractionEnabled)
                 {
-                    TeleportSystem.RaiseTeleportCanceled(eventData.Pointer, this);
-                    TeleportSystem.RaiseTeleportRequest(eventData.Pointer, this);
+                    MixedRealityManager.TeleportSystem.RaiseTeleportCanceled(eventData.Pointer, this);
```

Comment:
Not checking for no Teleport system
```
MixedRealityManager.TeleportSystem?.
```
Update and check throughout

Reply:
Fixed